<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuslineInfoDialog</class>
 <widget class="QDialog" name="BuslineInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>线路信息</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>14</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>16</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>线路编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTicket">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>82</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>票价信息：</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="firstTimeEidt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>46</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="lastTimeEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>46</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>首班车：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>末班车：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="addedStopsList">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>115</y>
     <width>141</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addFromListButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>添加-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFromListButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-移除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>452</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="allStopsList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>115</y>
     <width>140</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuslineInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuslineInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
